src/Pyz/Yves/HomePage/Theme/default/views/home/home.twig

```diff
@@ -28,3 +28,15 @@
         </main>
     </div>
 {% endblock %}
+
+{% block eventTracker %}
```

Comment:
the same Q, can we extend `https://github.com/spryker-shop/home-page/blob/master/src/SprykerShop/Yves/HomePage/Theme/default/views/home/home.twig` and don't include this block

Reply:
Updated